src/Surfnet/StepupMiddlewareClientBundle/Identity/Service/RaCandidateService.php

```diff
@@ -73,11 +73,12 @@ public function search(RaCandidateSearchQuery $query)
     /**
      * @param string $identityId
      * @param string $institution
+     * @param string $actorId
      * @return RaCandidateInstitutions
      */
-    public function get($identityId, $institution)
+    public function get($identityId, $institution, $actorId)
```

Comment:
This was not due to your change, but the 'use list' contains unused imports. Would you be so kind to remove them in this PR? #boyscout